<launch>
  <!-- params -->
  <param name="/simulator/scene_file" value="$(find pedsim_simulator)scenarios/demo_behaviours.xml" type="string"/>
  <param name="/pedsim/update_type" value="oracle" type="string"/>
  <param name="/pedsim/sensor_range" value="30.0" type="double"/>
  <param name="/pedsim/cell_size" value="1.0" type="double"/>

  <!-- other parameters -->
  <param name="/pedsim/cell_size" value="1.0" type="double"/>
  <param name="/pedsim/speed" value="1.0" type="double"/>
  <param name="/pedsim/target_type" value="2" type="int"/>

  <!-- robot control parameters -->
  <param name="/pedsim/move_robot" value="100" type="double"/>
  <param name="/pedsim/teleop_state" value="1.0" type="double"/>


  <!-- tf broadcaster for aligning frames -->
  <node name="simulator_tf_broadcaster" pkg="pedsim_simulator" type="dummy_transforms.py" respawn="false" output="log"
/>

  <!-- main simulator node -->
  <node name="simulator" pkg="pedsim_simulator" type="simulator" output="screen"/>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pedsim_simulator)/rviz/behaviour.rviz"/>


  <!-- Keyboard teleoperation node -->
  <node name="teleop" pkg="pedsim_simulator" type="teleop" respawn="false" output="log" />
</launch>
